frontend/src/components/OdhDocCard.tsx

```diff
@@ -34,46 +38,67 @@ const OdhDocCard: React.FC<OdhDocCardProps> = ({ odhDoc }) => {
     const label = getTextForDocType(odhDoc.metadata.type as ODHDocType);
     if (odhDoc.metadata.type === ODHDocType.Documentation) {
       return (
-        <>
-          <div className="odh-card__partner-badge odh-m-doc odh-m-documentation">{label}</div>
+        <div className="odh-card__doc-badges">
+          <Tooltip content="Technical information for using the service">
+            <div className="odh-card__partner-badge odh-m-doc odh-m-documentation">{label}</div>
+          </Tooltip>
           <div className="odh-card__partner-badge odh-m-doc m-hidden">N/A</div>
-        </>
+        </div>
       );
     }
     if (odhDoc.metadata.type === ODHDocType.Tutorial) {
       return (
-        <>
-          <div className="odh-card__partner-badge odh-m-doc odh-m-tutorial">{label}</div>
-          <div className="odh-card__partner-badge odh-m-doc">
+        <div className="odh-card__doc-badges">
+          <Tooltip content="End-to-end guides for solving business problems in data science">
+            <div className="odh-card__partner-badge odh-m-doc odh-m-tutorial">{label}</div>
+          </Tooltip>
+          <div className="odh-card__partner-badge odh-m-doc odh-m-duration">
             {odhDoc.spec.durationMinutes} minutes
           </div>
-        </>
+        </div>
       );
     }
     if (odhDoc.metadata.type === ODHDocType.QuickStart) {
-      const quickStart =
-        qsContext.allQuickStarts &&
-        qsContext.allQuickStarts.find((qs) => qs.metadata.name === odhDoc.metadata.name);
+      if (!qsContext.allQuickStarts) {
+        return null;
+      }
+      const quickStart = qsContext.allQuickStarts.find(
+        (qs) => qs.metadata.name === odhDoc.metadata.name,
+      );
       if (!quickStart) {
         return null;
       }
       return (
         <>
-          <div className="odh-card__partner-badge odh-m-doc odh-m-quick-start">{label}</div>
-          <div className="odh-card__partner-badge odh-m-doc">
-            {odhDoc.spec.durationMinutes} minutes
+          <div className="odh-card__doc-badges">
+            <Tooltip content="Step-by-step instructions and tasks">
+              <div className="odh-card__partner-badge odh-m-doc odh-m-quick-start">{label}</div>
+            </Tooltip>
+            <div className="odh-card__partner-badge odh-m-doc odh-m-duration">
+              {odhDoc.spec.durationMinutes} minutes
+            </div>
           </div>
+          {isQuickStartInProgress(quickStart.metadata.name, qsContext) ? (
+            <div className="odh-card__doc-badges .m-progress">
+              <div className="odh-card__partner-badge odh-m-doc odh-m-in-progress">
+                <SyncAltIcon />
+                In Progress
```

Comment:
To align more with the default quick starts catalog library, could be sentence case
```suggestion
                In progress
```

Default quick starts catalog as of now also displays the labels in the following order:
1. (Optional) type/category - i.e. Quick start / Documentation
2. In progress / Complete
3. Duration

Reply:
Are the horizontal? Could you attach a screen shot of a card with all three?

Reply:
At the moment it looks like this:
![Screen Shot 2021-03-23 at 3 35 12 PM](https://user-images.githubusercontent.com/869106/112207372-67ce1080-8bed-11eb-88cd-4f8e75f0fbd1.png)

But we can change it if it makes more sense in a different order.
cc @jgiardino

Reply:
I'll change the order to follow suit with your order:
1. Type
2. Duration
3. In progress / complete

Reply:
👍 I also like the order that RHODS currently uses.